src/main/app/claims/ccjModelConverter.ts

```diff
@@ -42,20 +42,23 @@ function convertPayBySetDate (draftCcj: DraftCCJ): Moment {
 }
 
 export function retrievePaymentOptionsFromClaim (claim: Claim): CCJPaymentOption {
-  if (claim.response && claim.isAdmissionsResponse() &&
-    ((claim.settlement && claim.settlementReachedAt) || claim.hasDefendantNotSignedSettlementAgreementInTime())) {
+  if (!claim.response || !claim.isAdmissionsResponse()) {
+    return undefined
+  } else if (((claim.settlement && (claim.settlementReachedAt || claim.settlement.isOfferRejectedByDefendant()))
+    || claim.hasDefendantNotSignedSettlementAgreementInTime())) {
```

Comment:
I think there are some extra parentheses here.
Haven't checked it myself but try `(claim.settlement && (claim.settlementReachedAt || claim.settlement.isOfferRejectedByDefendant()) || claim.hasDefendantNotSignedSettlementAgreementInTime())`

A common `isSettlementCompleted()` or some such could be handy